scss/civicrm/api/_api.scss

```diff
@@ -0,0 +1,141 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('api')} {
+  .crm-container {
+    #mainTabContainer {
```

Comment:
why are these two selectors necessary, if we are already namespacing the style via `#{civi-page('api')}`

Reply:
2 reasons,
1. when we have `.page-civicrm-api` [Generated from `#{civi-page('api')}`] it skips .crm-containter wrapper which we add to all style we do for civicrm. So, we follow that on all the pages.
2. This page contains inline style like below,
```css
#mainTabContainer pre {
  line-height: 14px;
  font-size: 11px;
  margin: 0;
  border: 0 none;
}
#mainTabContainer ul.ui-tabs-nav {
  font-size: 1.1em;
  margin-bottom: .6em;
}
 ......
```
So, we are making it to override it properly.